### PR TITLE
Alternative Round 1 w/ reset vs. Master branch

### PR DESCRIPTION
**Pull request by cthoyt**
I ran `git reset --soft master` so all of the changes apparent in https://github.com/CoronaWhy/task-vt/pull/41 are compared to master. Unfortunately, I wasn't able to undo @yangjunqingqiao's because I don't know the code well enough

@yangjunqingqiao you have to say if there are any things that didn't make it from the last PR here.

### CODE REVIEW

File: main_pipeline/data_utils/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# Utilities for dataverse or whatever we decide to use for data hosting will go here. Also putting data processing utils here
```
Comment (yangjunqingqiao) [inline main_pipeline/data_utils/__init__.py:1]: This file should be deleted (I forgot to in my branch as well but it's redundant now).

File: main_pipeline/tests/util_tests.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import unittest
```
Comment (yangjunqingqiao) [inline main_pipeline/tests/util_tests.py:1]: This and `main_pipeline/tests/test_utils/__init__.py` should be deleted as well.

File: src/coronawhy_vt/utils/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+"""Utilities for dataverse."""
```
Comment (yangjunqingqiao) [inline src/coronawhy_vt/utils/__init__.py:3]: Missing ending newline

File: tests/__init__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (yangjunqingqiao) [inline tests/__init__.py:1]: Also seems to be missing newline, strange that flake8 didn't catch this.

File: tests/test_utils_log.txt [hunk 5]
```diff
@@ -0,0 +1,5 @@
+............
```
Comment (yangjunqingqiao) [inline tests/test_utils_log.txt:1]: Realizing that log files should probably not be added to the repo

**Comment by cthoyt** [comment]
@yangjunqingqiao can we delete this PR and go back to the original one now? Or was there something else you wanted to look at?

**Comment by yangjunqingqiao** [comment]
I think it's probably safe to delete this one. I tested the thing I wanted to on this PR (changing the target branch).

**Pull request closed without merging**